src/lib/util/External/escapeMarkdown.ts

```diff
@@ -0,0 +1,156 @@
+/**
```

Comment:
After moving `escapeCodeBlock` to this file as suggested by Kyra be sure to export `escapeMarkdown` and `escapeCodeBlock` from this file then rename the file to `escapeText.ts` so it's more generic for its content.

Reply:
escapeMarkdown is already pretty generic.
Also, all functions should be exported.